packages/mapviewer/src/modules/menu/components/activeLayers/MenuActiveLayersListItem.vue

```diff
@@ -62,7 +62,7 @@ const emit = defineEmits(['showLayerDescriptionPopup', 'toggleLayerDetail', 'mov
 const store = useStore()
 const { t } = useI18n()
 
-useTippyTooltip('.menu-layer-item [data-tippy-content]')
+const menuLayerItem = useTemplateRef('menuLayerItem')
```

Comment:
That's huge, this new `useTemplateRef` from Vue 3.5.
We should move all our old `... = ref(null)` from before to this new notation

Reply:
created a ticket for that https://jira.swisstopo.ch/browse/PB-1428

Reply:
Yes, I didn't know that either! Makes it so much better